quarto/src/Api/GameManager.php

```diff
@@ -20,14 +20,14 @@ public function __construct(GameRepository $gameRepository)
         $this->gameRepository = $gameRepository;
     }
 
-    public function newGame(int $size, string $playerName = '') : Game
+    public function newGame(int $size, $playerName = '') : Game
```

Comment:
When a call is made without sending a playerName (with older versions of QuartoIso or QuartoSymfony), the function doesn't agree paramter because it's null, and string != null
Without typing it, null is accepted, and function keep backward compatibility